### ISSUE TITLE
Make Exists validator more generic to support different models

### ISSUE THREAD

**Issue by anxolerd** [uaprom-summer-2015/Meowth#182]
## Description
For now [`Exists` validator](https://github.com/uaprom-summer-2015/Meowth/blob/master/project/auth/validators.py#L33-L54) supports only `User` model. Make it support different models.

## Suggestions
I suggest to add model class as constructor' param [[here]](https://github.com/uaprom-summer-2015/Meowth/blob/master/project/auth/validators.py#L34):
```python
def __init__(self, model, message=None, reverse=False):
    ...
    self.model = model
    ...
```
and then use it instead of explicitly using `User` model:
```python
self.model.query.filter(getattr(self.model, field.name) == field.data)
```

**Comment by anxolerd** [comment]
Related #181

**Comment by akhalyavka** [comment]
Look #183

**Comment by Cybran111** [comment]
Fixed in #185
